Libraries/LibGfx/Bitmap.cpp

```diff
@@ -182,6 +182,28 @@ Bitmap::Bitmap(BitmapFormat format, NonnullRefPtr<SharedBuffer>&& shared_buffer,
         allocate_palette_from_format(m_format, palette);
 }
 
+RefPtr<Gfx::Bitmap> Bitmap::clone() const
+{
+    const auto w = this->width();
+    const auto h = this->height();
```

Comment:
No need to specify `this` all the time, it's implied from context :)
Also, you can use `Bitmap::size()` directly instead of making a temporary `IntSize`.